src/core/MasterClueCounter.ttslua

```diff
@@ -6,11 +6,27 @@ local clueCounterGUIDS = {
   "d86b7c"
 }
 
-function onLoad()
+local clueClickers = {}
+local clueClickerGUIDS = {
+  "db85d6",
+  "3f22e5",
+  "891403",
+  "4111de"
+}
+
+count = 0
+clickableCounters = false
```

Comment:
Naming: useClickableCounters.  Also, any reason not to make these local?

Reply:
Renaming it with next commit. It's not local because this will be changed when the option is toggled.